projects/common-lib/src/lib/language-switch/language-switch.component.html

```diff
@@ -1,7 +1,9 @@
 <div id="language-switch">
-  <div class="icon"><i class="fa fa-language"></i></div>
-  <div class="selected-language">
-    <span>{{ lang.currentLanguage.symbol }}</span>
+  <div class="selected-language"> 
+    <img class="flag-img" src="assets/icons/flags/{{ lang.currentLanguage.symbol }}.svg">
```

Comment:
Works good. But it will be better if we use the `lang.currentLanguage.code` instead. Because we can have american english, british english, australian english. Mexican spanish, Spain spanish, etc. 

So the flag icons will be named: `en_US.svg`, `es_ES.svg`

Reply:
Also, I think the flag icons must be in a remote server in order to be rendered in external applications. Maybe putting them in `canta.com/assets/icons/*.svg`?

Reply:
@gus-canya good idea. I'll put them at canya.io...